UI/FleetButton.cpp

```diff
@@ -436,6 +493,23 @@ std::shared_ptr<GG::Texture> FleetSizeIcon(unsigned int fleet_size, FleetButton:
     return texture_temp;
 }
 
+std::shared_ptr<GG::Texture> FleetBlockadedIcon(FleetButton::SizeType size_type) {
+    if (size_type == FleetButton::FLEET_BUTTON_NONE)
+        return nullptr;
+
+    std::string size_prefix = FleetIconSizePrefix(size_type);
+    if (size_type == FleetButton::FLEET_BUTTON_TINY)
+        size_prefix = "tiny-";
+
+    std::shared_ptr<GG::Texture> retval = ClientUI::GetClientUI()->GetTexture(ClientUI::ArtDir() / "icons" / "fleet" / (size_prefix + "blockade.png"), false);
+    glBindTexture(GL_TEXTURE_2D, retval->OpenGLId());
```

Comment:
I can't really answer that. I reproduced how the textures for the other fleet icons are loaded (FleetHeadIcons(), FleetSizeIcon(), FleetSelectionIndicatorIcon()).